content/en/running-programs/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Running CHT implementations"
+linkTitle: "CHT Implementations"
+weight : 2
+---
+
+There are various ways of running Community Health Toolkit implementations. It is important to understand the different models for integrating CHT tools into a community health program as discussed below.
+
+**Accompaniment approach**
+
+In this approach, Medic accompanies partners; including ministries of health, through their digital journey that may include designing, building, deploying and supporting CHT based tools for their unique health system needs. 
+
+**Community led implementations**
+
+This model leverages the robust technical expertise of the CHT community of practice. A network of local and regional technical organizations that have been onboarded on CHT are able to provide design, development, deployment and support services for implementing partners - including ministries of health. This network also contributes their learnings to the community through documentation, reference applications, CHT forum and CHT core improvements. Community led implementations emphasize on the importance of collaboration, local ownership and sustainability.
```

Comment:
I think we need to be really explicit that community led implementations can happen with no assistance from Medic or the release of the community. Currently this whole paragraph makes it feel like you need a whole lot of technical expertise to pull it off, or you need to work with the network of local technical orgs. How can we reword to make it clear this is a self-led deployment with community support available?